_scripts/tests/install.sh

```diff
@@ -22,6 +22,7 @@ if [ ! -z $CI ]; then
   echo "INJECTING VERSIONS!"
   echo "-------------------"
   pushd ../../ &> /dev/null
+  bumpver-if-set "deis-tests" "deis-e2e" "${WORKFLOW_E2E_SHA}"
```

Comment:
really tried to hold back... but how about `WORKFLOW_E2E_GIT_SHA` for consistency? :)